camelot/view/controls/editors/one2manyeditor.py

```diff
@@ -235,7 +235,10 @@ def trigger_list_action(self, index):
         table.close_editor()
         # make sure ChangeSelection action is executed before list action
         table.model().submit()
-        self._run_list_context_action(self, None)
+        obj_id = table.model().headerData(index, Qt.Orientation.Vertical, ObjectRole)
```

Comment:
Op windows geeft dit -1 terug asl obj_id. Waarschijnlijk omdat die data nog niet is opgehaald van de server...

Hier verschuiven we dus enkel het probleem.

Reply:
das toch wel weird ...